integration/benchmark/contLoadGenerator.go

```diff
@@ -286,6 +288,11 @@ func (lg *ContLoadGenerator) Stop() {
 	lg.workerStatsTracker.StopPrinting()
 	lg.log.Debug().Msg("stopping follower")
 	lg.follower.Stop()
+	close(lg.stoppedChannel)
```

Comment:
do we need to worry about double close?

Reply:
Adding a check to the beginning of stop to make sure we are safely stopping. We would have to make changes to Start() if we want a single load generator to be able to be stopped and started multiple times, right now the load generators are one time use (one init, one start, one stop, and then if we need it again we need a whole new load generator).  Not sure if there is a use case for using the same load generator multiple times but worth considering for the future.